recipes-kernel/linux/files/uos_rt.cfg

```diff
@@ -0,0 +1,17 @@
+CONFIG_ACRN_VIRTIO_DEVICES=y
+CONFIG_HYPER_DMABUF_ACRN=y
+CONFIG_ACRN=y
+CONFIG_PREEMPT_COUNT=y
+CONFIG_PREEMPT=y
+CONFIG_X86_X2APIC=y
+CONFIG_RCU_NOCB_CPU=y
+CONFIG_PREEMPT_RCU=y
+CONFIG_PREEMPT_RT_FULL=y
+CONFIG_PREEMPT_LAZY=y
+CONFIG_PREEMPT_RT_BASE=y
+CONFIG_NO_HZ_FULL=y
+CONFIG_NO_HZ_IDLE=n
+CONFIG_PREEMPT_NONE=n
+CONFIG_EARLY_PRINTK_EFI=y
```

Comment:
Debugging releated configuration no need to have as they can be enabled by user.

Reply:
agree, debugging features should be optional and can be enable through KERNEL_EXTRA_FEATURES or menuconfig